src/host/ut_host/ScreenBufferTests.cpp

```diff
@@ -2845,17 +2840,8 @@ void ScreenBufferTests::SetDefaultForegroundColor()
     newColor = gci.GetDefaultForegroundColor();
     VERIFY_ARE_EQUAL(testColor, newColor);
 
-    Log::Comment(L"Invalid Decimal Notation");

```

Comment:
There are actually valid right now. So I remove these. I try to leave the intensive tests to the s_ParseColorSpec method tests. So I think the public API tests can be simplified.